app/networkapi/settings.py

```diff
@@ -43,6 +43,7 @@
 
 
 ALLOWED_HOSTS = env('ALLOWED_HOSTS')
+CSRF_TRUSTED_ORIGINS = env('CSRF_TRUSTED_ORIGINS')
```

Comment:
if this is supposed to be an array, add it to the env conf above too to parse it as an array